Signal/src/call/PeerConnectionClient.swift

```diff
@@ -245,49 +282,55 @@ class PeerConnectionClient: NSObject, RTCPeerConnectionDelegate, RTCDataChannelD
 
     public func setLocalVideoEnabled(enabled: Bool) {
         SwiftAssertIsOnMainThread(#function)
+        let handle = self.handle
+        let completion = { [weak self] in
+            guard let strongSelf = handle.get() else { return }
+            guard let localVideoTrack = strongSelf.localVideoTrack else { return }
+            guard let strongDelegate = strongSelf.delegate else { return }
+            strongDelegate.peerConnectionClient(strongSelf, didUpdateLocal: enabled ? localVideoTrack : nil)
+        }
```

Comment:
Previously, we often defined blocks within blocks.  I didn't want to risk accidentally capturing strongSelf from the outer block in an inner block, so I've "unwrapped" these blocks to all be defined separately.  I guess it's worth saying that I didn't want to have `strongSelf1`, `strongSelf2` or whatever either.